chain/cosmos/ics.go

```diff
@@ -148,7 +157,7 @@ func (c *CosmosChain) StartProvider(testName string, ctx context.Context, additi
 			InitialHeight: clienttypes.Height{RevisionNumber: clienttypes.ParseChainID(consumer.cfg.ChainID), RevisionHeight: 1},
 			GenesisHash:   []byte("gen_hash"),
 			BinaryHash:    []byte("bin_hash"),
-			SpawnTime:     time.Now(), // Client on provider tracking consumer will be created as soon as proposal passes
+			SpawnTime:     time.Now().Add(2 * time.Minute), // Needed so that there's time for consumer keys to be assigned
```

Comment:
Does it take 2 minutes to get keys copied?

Maybe instead something like X seconds * number of chains you are starting or something? this way it scales with complexity instead of a hardcoded value. Seems long for a simple ICS <-> Consumer each with 1 val

or maybe an ENV_VAR to override this to make it longer as needed? (would need to add to the docs/env*.md section)

Reply:
yeah, I like the env var solution